contracts/axelar-soroban-interfaces/src/types.rs

```diff
@@ -1,5 +1,4 @@
 use soroban_sdk::{contracttype, xdr::ToXdr, Address, BytesN, Env, String, Vec};
-
```

Comment:
why remove the newline

Reply:
I think it was removed while add/remove dependencies by mistake. updated.